CMakeLists.txt

```diff
@@ -22,7 +22,20 @@ include(CheckCSourceCompiles)
 include(CheckIncludeFile)
 include(CheckCCompilerFlag)
 include(CMakePackageConfigHelpers)
+include(CMakePushCheckState)
 include(GNUInstallDirs)
+include("${CMAKE_CURRENT_LIST_DIR}/cmake/sdlchecks.cmake")
+
+CheckOpenGLES()
+
+cmake_push_check_state()
+set(CMAKE_REQUIRED_FLAGS "-Wl,--no-undefined")
```

Comment:
Note: SDL autofoo and cmake'ry blacklists OpenBSD for this.

Reply:
I found this: https://github.com/libsdl-org/SDL/issues/3720
So let's disable it there again.

Reply:
SDL does only add `-Wl,--no-undefined` for non-Apple non-OpenBSD platforms.
Shouldn't apple also get it?

Reply:
For Apple's ld, we use `-Wl,-undefined,error` (someone recently removed it for reasons I don't fully understand.)

Reply:
https://github.com/libsdl-org/SDL/pull/6365
It's redundant, he says.

Reply:
Hm. Ookay. (I still have it in my standalone Makefile.darwin)